src/queries.ts

```diff
@@ -1,4 +1,5 @@
 import { visit, namedTypes as n } from 'ast-types';
+import { get } from 'lodash';
```

Comment:
do we need all lodash, maybe better to used polyfills? the library would have a smaller size

Reply:
In the future will be more libraries in this file from loads. So currently is not top prior. :)

Reply:
👍